lib/matplotlib/widgets.py

```diff
@@ -3968,16 +3856,17 @@ class Lasso(AxesWidget):
         The parent Axes for the widget.
     xy : (float, float)
         Coordinates of the start of the lasso.
+    callback : callable
+        Whenever the lasso is released, the *callback* function is called and
+        passed the vertices of the selected path.
     useblit : bool, default: True
         Whether to use blitting for faster drawing (if supported by the
         backend). See the tutorial :doc:`/tutorials/advanced/blitting`
         for details.
-    callback : callable
-        Whenever the lasso is released, the *callback* function is called and
-        passed the vertices of the selected path.
     """
 
-    def __init__(self, ax, xy, callback=None, useblit=True):
+    @_api.make_keyword_only("3.7", name="useblit")
+    def __init__(self, ax, xy, callback, useblit=True):
```

Comment:
`callback` cannot be None, so it doesn't make sense to have a default value (despite this going back 17 years...).

Reply:
I'm a bit skeptical of changing callback to be keyword only.  The only place we use this in a demo it is passed positionally.  I am worried that we are going to get minor payoff at the cost of thrashing a bunch of user code.

Reply:
I would not make it kwonly; it should be pretty clear in the context that that argument is a callback.  The point of making e.g. useblit kwonly is that if you write `Lasso(ax, xy, somefunc, True)`, it is really not clear what `True` is (`useblit=True` is clearer), but it is quite obvious that somefunc is a callback (especially as it'll typically have a more meaningful name, e.g. onselect; if it's a lambda defined inline (`Lasso(ax, xy, lambda vertices: ...)`) the situation is also clear).